modules/chezmoi/README.md

```diff
@@ -0,0 +1,59 @@
+# `chezmoi`
+
+The `chezmoi` module takes care of installing, initializing and updating your dotfiles.
+Each feature can be enabled or disabled individually.
+
+Installation of the `chezmoi` binary happens at build time and is done by downloading the `amd64` binary from the latest release to `/usr/bin/chezmoi`. 
+This can be disabled by setting `install` to false. (defaults: true)
+
+A systemd user service is installed that will initialize a `chezmoi` repository on chezmoi's default path (`~/.local/share/chezmoi`) for any user when it logs in, or at boot if it has lingering enabled.
+The service will only run if `~/.local/share/chezmoi` does not exist.
+Set `repository` to the URL of your dotfiles repository. (eg. `repository: https://example.org/user/dotfiles`)
+:::note
+The value of `repository` will be passed directly to `chezmoi init --apply ${repository}`.
+See the [`chezmoi init` documenation](https://www.chezmoi.io/reference/commands/init/) for detailed syntax.
+::: 
+Set `disable_init` to `true` if you do not want to install the init service.
+
+:::caution
+If `repository` is not set, and `disable_init` is false the module will fail, due to not being able to initialize the repository.
+:::
+
+Set `enable_all_users` to `false` if you want to install the update and initialization services, but do not want them enabled for all users.
+You can enable them manually instead when the system has been installed:
+
+To enable the services for a single user, run the following command as that user:
+
+```bash
+systemctl enable --user chezmoi-init.service chezmoi-update.timer`
+```
+
+To manually enable the services for all users, run the following command with sudo:
+
+```bash
+sudo systemctl enable --user chezmoi-init.service chezmoi-update.timer
+```
+
+To turn on lingering for a given user, run the following commmand with sudo:
```

Comment:
Should explain what lingering is (and possible security concerns).

Reply:
Did this in: 1bb33dc381312b44ec0c25dff5905db17f1efcb1
I would like to have an example in the docs of when it would make sense to enable lingering, but i can't think of a use case.

I couldn't find nor think of any security concerns. It shouldn't have any as long as, a user being logged in or not, is not used to start and stop sensitive services, which would be strange. 
Please correct me if anyone knows something i don't!